app/controllers/categories_controller.rb

```diff
@@ -135,10 +135,25 @@ def show
     render_serialized(@category, CategorySerializer)
   end
 
+  MAX_DESCRIPTION_PARAM_LENGTH = 1000
   def create
     guardian.ensure_can_create!(Category)
     position = category_params.delete(:position)
 
+    if category_params[:description].present? &&
+         category_params[:description].bytesize > MAX_DESCRIPTION_PARAM_LENGTH
```

Comment:
Why `.bytesize` and not `.size` / `.length`? 🤔 

Most length checks we do are based on "characters" rather than "bytes", I think?